Tools-Override/codeAnalysis.targets

```diff
@@ -46,7 +42,7 @@
     <AdditionalFiles Condition="'$(ValidPinvokeMappings)' != ''" Include="$(ValidPinvokeMappings)"/>
 
     <!-- Disable any analyzers that should not run -->
-    <DisabledAnalyzers Condition="'$(IsTestProject)' == true or '$(TargetsUnix)' == 'true'" Include="PinvokeAnalyzer"/>
```

Comment:
I think it is worth having a DisablePInvokeAnalyzer property so projects have a way to turn this off completely if necessary.